projects/ngx-d3timeline/src/lib/ngx-d3timeline.service.ts

```diff
@@ -36,8 +49,14 @@ export class NgxD3TimelineService implements OnDestroy {
   resourceTickMarkRectangles$ = this.store.select(selectResourceTickRectangles);
 
   private destroySubject = new Subject<boolean>();
-
-  constructor(private store: Store, private axisService: AxisService) {}
+  constructor(
+    private store: Store,
+    private axisService: AxisService,
+    hostElement: ElementRef,
+    zone: NgZone
```

Comment:
this will have to be made a field `private zone: NgZone` to move `this.setupResizing` into the onInit of constructor.

Reply:
done